services/minespace-web/src/components/Forms/mines/EditWorkerInformationForm.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Form, Button, Col, Icon, Popconfirm, Row, Tooltip, Typography, Descriptions } from "antd";
+
+import { Field, reduxForm } from "redux-form";
+import { integer, required } from "@common/utils/Validate";
+import { renderConfig } from "@/components/common/config";
+import * as FORM from "@/constants/forms";
+
+const { Title } = Typography;
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.objectOf(PropTypes.any)]).isRequired,
+  submitting: PropTypes.bool.isRequired,
+  handleToggleEdit: PropTypes.func.isRequired,
+};
+
+export const EditWorkerInformationForm = (props) => (
+  <div className="work-information-container ">
+    <Form layout="vertical" onSubmit={props.handleSubmit}>
+      <Title level={4}>
+        Worker Information
+        <Tooltip
+          overlayClassName="minespace-tooltip"
+          title="Approximate number of workers on site that include mine employees and contractors."
+          placement="right"
+          mouseEnterDelay={0.3}
+        >
+          <Icon type="info-circle" className="padding-sm" />
+        </Tooltip>
+      </Title>
+      <Row gutter={16}>
+        <Col span={22}>
+          <Descriptions column={5} colon={false}>
+            <Descriptions.Item label="Number of Mine Employees">
+              <Form.Item>
+                <Field
+                  id="number_of_mine_employees"
+                  name="number_of_mine_employees"
+                  component={renderConfig.FIELD}
+                  validate={[integer, required]}
+                  format={null}
+                />
+              </Form.Item>
+            </Descriptions.Item>
+            <Descriptions.Item label="Number of Contractors">
+              <Form.Item>
+                <Field
+                  id="number_of_contractors"
+                  name="number_of_contractors"
+                  component={renderConfig.FIELD}
+                  validate={[integer, required]}
+                  format={null}
+                />
+              </Form.Item>
+            </Descriptions.Item>
+          </Descriptions>
+        </Col>
+      </Row>
+      <div className="right center-mobile">
+        <Popconfirm
+          placement="topRight"
+          title="Are you sure you want to cancel?"
+          onConfirm={(event) => {
+            props.handleToggleEdit(event);
+          }}
+          okText="Yes"
+          cancelText="No"
+          disabled={props.submitting}
+        >
+          <Button className="full-mobile margin-small" type="secondary" disabled={props.submitting}>
+            Cancel
+          </Button>
+        </Popconfirm>
+        <Button
+          className="full-mobile margin-small"
+          type="primary"
+          htmlType="submit"
+          loading={props.submitting}
+        >
```

Comment:
can you also add `disabled={props.submitting}`

Reply:
Thanks :)